### ISSUE TITLE
'Axis=1' in Concatenate cause conflict in dimension of logits and labels.

### ISSUE THREAD

**Issue by Vikas-Sehwag** [DT42/squeezenet_demo#8]
merge_i = Concatenate(axis=1)(a,b)

1) It should be 'axis=-1'.

**Comment by qjfytz** [comment]
cannot agree. The data is in the format 'channel first', which means the channels placed in the axis 1.
